assets/js/modules/analytics/datastore/constants.js

```diff
@@ -37,4 +37,6 @@ export const EDIT_SCOPE = 'https://www.googleapis.com/auth/analytics.edit';
 // Date range offset days for Analytics report requests.
 export const DATE_RANGE_OFFSET = 1;
 
-export const FORM_ALL_TRAFFIC_WIDGET = 'analyticsAllTrafficWidget';
+export const DIMENSION_NAME_ALL_TRAFFIC_WIDGET = 'dashboardAllTrafficWidgetDimensionName';
+export const DIMENSION_COLOR_ALL_TRAFFIC_WIDGET = 'dashboardAllTrafficWidgetDimensionColor';
+export const DIMENSION_VALUE_ALL_TRAFFIC_WIDGET = 'dashboardAllTrafficWidgetDimensionValue';
```

Comment:
See my other comment, how about renaming the constants to `UI_DIMENSION_NAME`, `UI_DIMENSION_COLOR`, `UI_DIMENSION_VALUE`?

Reply:
Sounds good. Updated @felixarntz